cli/commands/preview/list.ts

```diff
@@ -47,14 +59,21 @@ export const registerCommand = ( yargs: StudioArgv ) => {
 		command: 'list [folder]',
 		describe: __( 'List preview sites for the specified folder (defaults to current directory)' ),
 		builder: ( yargs ) => {
-			return yargs.positional( 'folder', {
-				type: 'string',
-				default: process.cwd(),
-				description: __( 'The folder to list previews for' ),
-			} );
+			return yargs
+				.positional( 'folder', {
+					type: 'string',
+					default: process.cwd(),
+					description: __( 'The folder to list previews for' ),
```

Comment:
I wonder if it would make sense to make the following changes as well:

```suggestion
					description: __( 'The folder to list preview sites for' ),
```